files/zh-cn/web/api/domtokenlist/toggle/index.md

```diff
@@ -4,58 +4,76 @@ slug: Web/API/DOMTokenList/toggle
 ---
 {{APIRef("DOM")}}
 
-{{domxref("DOMTokenList")}} 接口的 **`toggle()`** 方法从列表中删除一个给定的*标记* 并返回 `false` 。 如果*标记* 不存在，则添加并且函数返回 `true`。
+{{domxref("DOMTokenList")}} 接口的 **`toggle()`** 方法从列表中删除一个给定的标记并返回 `false`。如果标记不存在，则添加并且函数返回 `true`。
 
 ## 语法
 
-```plain
-tokenList.toggle(token, force);
+```js
+toggle(token);
+toggle(token, force);
 ```
 
-### 参数列表
+### 参数
 
 - token
-  - : 标记列表中你想探查并切换的 {{domxref("DOMString")}} .
+  - : 标记列表中你想探查并切换的 {{domxref("DOMString")}}。
 - force {{optional_inline}}
-  - : 一个{{domxref("Boolean")}} 值，设置后会将方法变成单向操作。如设置为`false`, 则会删除标记列表中匹配的给定标记，且不会再度添加。如设置为 `true`, 则将在标记列表中添加给定标记，且不会再度删除。
+  - : 一个 {{domxref("Boolean")}} 值，设置后会将方法变成单向操作。如设置为 `false`，则会删除标记列表中匹配的给定标记，且不会再度添加。如设置为 `true`，则将在标记列表中添加给定标记，且不会再度删除。
+
+### 参数
+
+- `token`
+  - : 一个字符串，表示你想要 toggle 的标记。
+- `force` {{optional_inline}}
+  - : 如果包含该值，设置后会将方法变成单向操作。如果设置为 `false`，*仅*会删除标记列表中匹配的给定标记，且不会再度添加。如果设置为 `true`，*仅*在标记列表中添加给定标记，且不会再度删除。
 
 ### 返回值
 
-该方法返回一个{{domxref("Boolean")}}值 — 如给定标记不存在于列表中返回`false` , 标记存在则返回`true` 。
+一个布尔值，`true` 或者 `false`，表示在调用 toggle 后，标记是否存在与列表中。
 
-## Examples
+## 示例
 
-In the following example we retrieve the list of classes set on a {{htmlelement("span")}} element as a `DOMTokenList` using {{domxref("Element.classList")}}. We then replace a token in the list, and write the list into the `<span>`'s {{domxref("Node.textContent")}}.
+### 在点击后 toggle class
 
-First, the HTML:
+以下示例我们使用 {{domxref("Element.classList")}} 检索 {{htmlelement("span")}} 元素的 class 集合列表（其返回值为 `DOMTokenList`）。然后我们替换集合中的一个标记，并将集合写入 `<span>` 的 {{domxref("Node.textContent")}}
+
+这是 HTML：
 
 ```html
 <span class="a b">classList is 'a b'</span>
 ```
 
-Now the JavaScript:
+现在是 JavaScript：
 
 ```js
-var span = document.querySelector("span");
-var classes = span.classList;
-span.onclick = function() {
-  var result = classes.toggle("c");
-  if(result) {
-    span.textContent = "'c' added; classList is now '" + classes + "'.";
+const span = document.querySelector("span");
+const classes = span.classList;
+span.addEventListener('click', () => {
+  const result = classes.toggle("c");
+  if (result) {
+    span.textContent = `'c' added; classList is now "${classes}".`;
   } else {
-    span.textContent = "'c' removed; classList is now '" + classes + "'.";
+    span.textContent = `'c' removed; classList is now "${classes}".`;
   }
-}
+})
 ```
 
-The output looks like this:
+输出如下所示，并且每次点击文本时都会发生变化：
+
+{{ EmbedLiveSample('Toggling_a_class_on_click', '100%', 60) }}
```

Comment:
please see <https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Live_samples>. You should update the param when translate those head. It's also metioned in [translation-guide](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md)

```suggestion
{{ EmbedLiveSample('在点击后 toggle class', '100%', 60) }}
```

Reply:
I see,thank you